cranelift/isle/isle/Cargo.toml

```diff
@@ -8,12 +8,17 @@ readme = "../README.md"
 repository = "https://github.com/bytecodealliance/wasmtime/tree/main/cranelift/isle"
 version = "0.107.0"
 
+[[test]]
+name = "printer_tests"
+required-features = ["printer"]
```

Comment:
Under this implementation, I believe the printer tests would not be run by default in CI.

It would be nice if cargo supported some kind of "default test features" configuration, but as far as I can tell this isn't possible. Nor is there much appetite for adding it according to https://github.com/rust-lang/cargo/issues/2911.

Perhaps if we cared enough we could edit the Github actions workflows to invoke these tests with the right `--feature` argument. But having poked around the CI config a little, it's not clear how to do that without causing a mess.

Another option is to have the printer tests in a different crate that can depend on this one with the right feature set.

All that said, I'm not sure it's worth it just for these tests, but I wanted to mention it in case you had a simple suggestion for how to run them?